plugin/session_buffer.py

```diff
@@ -185,14 +184,14 @@ def on_diagnostics_async(self, raw_diagnostics: List[Dict[str, Any]], version: O
                     data.panel_contribution.append(format_diagnostic_for_panel(diagnostic))
                 if diagnostic.severity <= settings.auto_show_diagnostics_panel_level:
                     should_show_diagnostics_panel = True
-            self._publish_diagnostics_to_session_views(
-                diagnostics_version,
-                diagnostics,
-                data_per_severity,
-                total_errors,
-                total_warnings,
-                should_show_diagnostics_panel
-            )
+        self._publish_diagnostics_to_session_views(
+            version,
+            diagnostics,
+            data_per_severity,
+            total_errors,
+            total_warnings,
+            should_show_diagnostics_panel
+        )
```

Comment:
Now that we know the actual problem we should revert this. Looks like this could result in another issue where we would show diagnostics but with all counts being 0.